commands/netobserv

```diff
@@ -72,7 +70,56 @@ function packets() {
   esac
 }
 
-case "$1" in  
+function compare_versions() {
+    IFS="." read -r -a ver1 <<< "$1"
+    IFS="." read -r -a ver2 <<< "$2"
+
+    for ((i = 0; i < ${#ver1[@]} || i < ${#ver2[@]}; i++)); do
+        v1=${ver1[i]:-0} # Default to 0 if unset
+        v2=${ver2[i]:-0}
+        if ((v1 < v2)); then
+            result=0 # less than
+            return
+        elif ((v1 > v2)); then
+            result=1 # greater than
+            return
+        fi
+    done
+    result=2 # equal
+}
+
+# Required version of yq
+REQUIRED_YQ_VERSION="v4.43.1"
```

Comment:
I there an interest to have this version different than the one defined in the Makefile ?
Else we can pass it as an argument. Check make `commands` target that calls the `inject.sh`